git_machete/client.py

```diff
@@ -1555,13 +1554,13 @@ def handle_untracked_branch(self, new_remote: str, branch: str, is_called_from_t
                 if ans in ('y', 'yes'):
                     self.__git.push(new_remote, branch)
                     self.flush_caches()
+                    return
                 elif can_pick_other_remote and ans in ('o', 'other'):
                     self.__pick_remote(branch, is_called_from_traverse)
                 elif ans in ('q', 'quit'):
                     raise StopInteraction
                 else:
                     raise MacheteException(f'Cannot create pull request from untracked branch `{branch}`')
-                return
```

Comment:
Whoops... actually, for the case of `can_pick_other_remote and ans in ('o', 'other'):`, should we `return` the entire method `handle_untracked_branch` after `__pick_remote` or not 🤔 it seems the behavior is different b/w `is_called_from_traverse` and `not is_called_from_traverse` cases now